### PR TITLE
osd/PrimaryLogPG: drop unused parameters

### PR DESCRIPTION
**Pull request by wzliao**
osd/PrimaryLogPG: drop unused parameters

### CODE REVIEW

**Comment by wzliao** [comment]
@joscollin , thank you for your notice, it has been changed now, please check it out.

**Comment by wzliao** [comment]
@joscollin , i put on the '-Wunused-parameter' option when build the project, got thousands '-Wunused-parameter' warnings, some of them should remain what they are for capability, some of them can be changed, but can not be easily found out, i think we should let they be.

File: src/osd/PrimaryLogPG.cc [hunk 1]
```diff
@@ -12536,7 +12536,7 @@ void PrimaryLogPG::hit_set_persist()
   map <string, bufferlist> attrs;
   attrs[OI_ATTR].claim(boi);
   attrs[SS_ATTR].claim(bss);
-  setattrs_maybe_cache(ctx->obc, ctx.get(), ctx->op_t.get(), attrs);
+  setattrs_maybe_cache(ctx->obc,  ctx->op_t.get(), attrs);
```
Comment (joscollin) [inline src/osd/PrimaryLogPG.cc]: Nit: Fix the unwanted space after `ctx->obc,`

Comment (wzliao) [reply src/osd/PrimaryLogPG.cc]: @joscollin , fixed it

**Comment by tchaikov** [comment]
http://pulpito.ceph.com/kchai-2017-10-31_13:59:11-rados-wip-kefu-testing-2017-10-31-1919-distro-basic-mira/

failed test tracked at http://tracker.ceph.com/issues/21993

**Pull request merged**